### DISCUSSION TITLE
加入希腊语输入法之后fcitx5托盘图标显示为错误

### DISCUSSION THREAD

**Discussion by CicadaSeventeen** [fcitx/fcitx5#869] [Q&A] [answered]
在只有英语键盘和拼音输入法时，输入法切换为英文时，托盘图标是正常显示的：
![image](https://github.com/fcitx/fcitx5/assets/86961676/ed3f1bb4-f4c3-4a1a-ab7e-0617a9102598)
但当我加入希腊语输入法之后，图标就变为了错误：
![image](https://github.com/fcitx/fcitx5/assets/86961676/58d3e7c0-5724-4a0e-82d3-62b7fd71d0e8)
此时，无论切换英文键盘还是希腊语键盘，图标都显示为错误。但切换为拼音输入法时，图标是正常的红色“拼”字

当然，这一问题不局限于希腊语，添加任何一种额外的键盘都会导致这类现象。当然，它不影响正常使用，但请问这可能是由什么导致的呢？

**Comment by wengxt** [comment, marked as answer]
你的托盘 SNI 实现不支持 IconPixmap

在布局超过两个的时候，将会使用布局的对应字母来显示布局

如图
![图片](https://github.com/fcitx/fcitx5/assets/259684/48c4fc57-608d-4a43-8f81-a708c0cb8bb2)

你可以在经典用户界面的配置里禁用这个选项

![图片](https://github.com/fcitx/fcitx5/assets/259684/e3542884-0f9d-45f9-be96-adb6f3613a09)

**Reply by CicadaSeventeen** [reply]
谢谢大佬，我传图片的时候传错了，两张都是正常的....抱歉
